draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -377,7 +377,8 @@ Inputs:
                  operation.
 - PK (REQUIRED), an octet string of the form outputted by the SkToPk
                  operation provided the above SK as input.
-- header (OPTIONAL), an octet string containing context and application
+- header (OPTIONAL), an octet string containing protected associated data 
+                     such as context and application
```

Comment:
IMO, those descriptions are not really suited for the input section of the operations, where we should try to keep things as simple as possible. I think there are 2 alternatives:
1. Add the `header` and `presentation_header` terms in the [Terminology section](https://identity.foundation/bbs-signature/draft-irtf-cfrg-bbs-signatures.html#name-terminology), followed by the above description.
2.  Add a new subsection of the [Considerations section](https://identity.foundation/bbs-signature/draft-irtf-cfrg-bbs-signatures.html#name-considerations), to describe their functionality in more detail.

(I would go with option 2)

This is in addition to the privacy section around the header/ph use, mentioned [here](https://github.com/decentralized-identity/bbs-signature/pull/255#issuecomment-1502141929)

Reply:
I like your options 2. Cheers Greg